tests/builtins/test_sum.py

```diff
@@ -46,6 +51,38 @@ class BuiltinSumFunctionTests(BuiltinFunctionTestCase, TranspileTestCase):
         'test_set',  # This works, but python dict.keys() returns non-deterministically.
     ]
 
-    not_implemented = [
-        'test_range',  # This has been implemented, but fails upstream on isinstance.
+
+class BuiltinSumTwoArgFunctionTests(BuiltinTwoargFunctionTestCase, TranspileTestCase):
+    function = "sum"
+
+    # these are implemented, but the exact exception thrown depends on the order
+    # that they are iterated on being the exact same in both CPython and batavia,
+    # which is not guaranteed. (if an unsupported string follows an int, the error
+    # will be different than if it followed a float)
```

Comment:
This is a familiar problem; historically, it's what we've used the "substitutions" feature of the test tools to fix. If you enumerate all possible orderings, and then use substitution to normalize to a consistent ordering, these tests can be made to pass.

Reply:
Good call. Updated.